common/changes/office-ui-fabric-react/aria-owns-bug-6931_2018-12-19-20-30.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fix for Issue#6931 : Datepicker:ARIA attributes must conform to valid values",
```

Comment:
nit: You might consider changing the formatting of the change file to something like the following:

ComponentName: description of fix [#issuenumber]

For example:

"DatePicker: only set aria-owns when callout is rendered [#6931]"